env/requirements.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
   version: 0.1.12
 - name: jx-app-ui
   repository: http://chartmuseum.jenkins-x.io
-  version: 0.1.211
+  version: 0.1.213
```

Comment:
since we don't automatically publish jx-app-ui releases in the public repo we'd need to use `https://chartmuseum-jx.jenkins-x.live`. I'd recommend to do that first and make sure it works before releasing in the public repo.
@cagiti WDYT?

Reply:
sure